napari/_tests/test_notebook_display.py

```diff
@@ -37,17 +37,23 @@ def test_nbscreenshot(make_napari_viewer):
         ("Good alt text", "Good alt text"),
         # Naughty strings https://github.com/minimaxir/big-list-of-naughty-strings
         # ASCII punctuation
-        (r",./;'[]\-=", ',./;&#x27;[]\\-='),
+        (r",./;'[]\-=", ",./;&#x27;[]\\-="),
         # ASCII punctuation 2, skipping < because that is interpreted as the start
         # of an HTML element.
-        ('>?:"{}|_+', '&gt;?:&quot;{}|_+'),
-        ("!@#$%^&*()`~", '!@#$%^&amp;*()`~'),  # ASCII punctuation 3
+        ('>?:"{}|_+', "&gt;?:&quot;{}|_+"),
+        ("!@#$%^&*()`~", "!@#$%^&amp;*()`~"),  # ASCII punctuation 3
         # # Emojis
         ("😍", "😍"),  # emoji 1
-        ("👨‍🦰 👨🏿‍🦰 👨‍🦱 👨🏿‍🦱 🦹🏿‍♂️", "👨‍🦰 👨🏿‍🦰 👨‍🦱 👨🏿‍🦱 🦹🏿‍♂️"),  # emoji 2
-        (r"¯\_(ツ)_/¯", '¯\\_(ツ)_/¯'),  # Japanese emoticon
+        (
+            "👨‍🦰 👨🏿‍🦰 👨‍🦱 👨🏿‍🦱 🦹🏿‍♂️",
+            "👨‍🦰 👨🏿‍🦰 👨‍🦱 👨🏿‍🦱 🦹🏿‍♂️",
+        ),  # emoji 2
+        (r"¯\_(ツ)_/¯", "¯\\_(ツ)_/¯"),  # Japanese emoticon
         # # Special characters
-        ("田中さんにあげて下さい", "田中さんにあげて下さい"),  # two-byte characters
+        (
+            "田中さんにあげて下さい",
+            "田中さんにあげて下さい",
+        ),  # two-byte characters
```

Comment:
Likely caused by this: https://docs.astral.sh/ruff/formatter/black/#line-width-vs-line-length